src/admin-views/aggregator/settings.php

```diff
@@ -364,6 +364,38 @@
 
 $internal = apply_filters( 'tribe_aggregator_fields', $internal );
 
+// fix this shit
+
+
+
+if ( Tribe__Events__Aggregator::instance()->is_service_active() ) {
+	$import_instructions = __( 'Use the options below to configure your imports. Global Import Settings apply to all imports, but you can also override the global settings by adjusting the origin-specific options.', 'the-events-calendar' );
```

Comment:
Should probably switch to `esc_html__()` like you've used elsewhere in this block.